amd_build/build_pytorch_amd.py

```diff
@@ -0,0 +1,47 @@
+import shutil
+import subprocess
+import os
+
+cwd = os.getcwd()
+proj_dir = os.path.dirname(cwd)
+out_dir = os.path.join(os.path.dirname(proj_dir), "pytorch_amd")
+exclude_dirs = [
+    "aten/src/TH",
+    "aten/src/THNN",
+    "aten/src/THS",
+    "caffe2",
+    "third_party"
+]
+
+yaml_file = os.path.join(cwd, "disabled_features.yaml")
+
+# Create the pytorch_amd directory
+shutil.copytree(proj_dir, out_dir)
+
+# Extract (.hip) files.
+shutil.copy(os.path.join(cwd, "hip_files/ATen/CMakeLists.txt.hip"), os.path.join(out_dir, "aten/src/ATen/CMakeLists.txt"))
+shutil.copy(os.path.join(cwd, "hip_files/_aten/CMakeLists.txt.hip"), os.path.join(out_dir, "aten/CMakeLists.txt"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/THCApply.cuh.hip"), os.path.join(out_dir, "aten/src/THC/THCApply.cuh"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/THCAsmUtils.cuh.hip"), os.path.join(out_dir, "aten/src/THC/THCAsmUtils.cuh"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/THCBlas.cu.hip"), os.path.join(out_dir, "aten/src/THC/THCBlas.cu.hip"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/THCDeviceUtils.cuh.hip"), os.path.join(out_dir, "aten/src/THC/THCDeviceUtils.cuh"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/THCNumerics.cuh.hip"), os.path.join(out_dir, "aten/src/THC/THCNumerics.cuh"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/THCTensorRandom.cu.hip"), os.path.join(out_dir, "aten/src/THC/THCTensorRandom.cu"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/THCTensorRandom.h.hip"), os.path.join(out_dir, "aten/src/THC/THCTensorRandom.h"))
+shutil.copy(os.path.join(cwd, "hip_files/THC/generic/THCTensorRandom.cu.hip"), os.path.join(out_dir, "aten/src/THC/generic/THCTensorRandom.cu"))
+
+# Move to avoid HCC bug.
+shutil.move(os.path.join(out_dir, "aten/src/ATen/native/cudnn/Conv.cpp"), os.path.join(out_dir, "aten/src/ATen/native/cudnn/ConvCuDNN.cpp"))
+shutil.move(os.path.join(out_dir, "aten/src/ATen/native/cudnn/BatchNorm.cpp"), os.path.join(out_dir, "aten/src/ATen/native/cudnn/BatchNormCuDNN.cpp"))
+
+# Apply the Patch File.
+subprocess.Popen(["git", "apply", os.path.join(cwd, "patch084e3a7.patch")], cwd=out_dir)
+
+# Execute the Hipify Script.
+subprocess.Popen(
+    ["/opt/rocm/bin/hipify-python.py",
```

Comment:
Most people aren't going to have this script. How do I get it?

Reply:
It's going to be on the HIP Codebase here: https://github.com/ROCm-Developer-Tools/pyHIPIFY, obtained from installing HIP/ROCm packages.

Reply:
OK, don't tell me, put it in the code so other people can find it :)

Reply:
@Jorghi12 Hi, I am trying to build your pytorch sending_pr branch for AMD but it seams the current pyHIPIFY is not up to date, option --include-dirs missing and I get the following error:
```
Total number of replaced kernel launches: 1433
Traceback (most recent call last):
  File "/opt/rocm/bin/hipify-python.py", line 845, in <module>
    main()
  File "/opt/rocm/bin/hipify-python.py", line 809, in main
    txt = disable_function(txt, func, 1)
  File "/opt/rocm/bin/hipify-python.py", line 329, in disable_function
    "return_type": the_match.group(2).strip(),
AttributeError: 'NoneType' object has no attribute 'group'
```
and is it ok to get the following error while patching?:
```
error: patch failed: aten/src/THC/THCTensorRandom.cpp:83
error: aten/src/THC/THCTensorRandom.cpp: patch does not apply
```